csi.proto

```diff
@@ -456,6 +456,17 @@ message ListVolumesResponse {
 message GetCapacityRequest {
   // The API version assumed by the CO. This is a REQUIRED field.
   Version version = 1;
+
+  // If specified, the Plugin SHOULD only report the capacity of the
```

Comment:
Answering my own question: some plugins may have no way of determining capacity. Anything else?